torchsparse/nn/functional/convert_neighbor_map.py

```diff
@@ -2,26 +2,13 @@
 import torchsparse_backend
 from torch.autograd import Function
 
+__all__ = ['convert_neighbor_map_gpu']
 
-class ConvertNeighborMap(Function):
-    @staticmethod
-    def forward(ctx, neighbor_map):
-        idx_batch, idx_point = torch.where(neighbor_map != -1)
-        if 'cuda' in str(neighbor_map.device):
-            map_converted = torchsparse_backend.convert_map_forward(
-                neighbor_map.int(), idx_batch.int(), idx_point.int())
-        elif 'cpu' in str(neighbor_map.device):
-            map_converted = torchsparse_backend.cpu_convert_map_forward(
-                neighbor_map.int(), idx_batch.int(), idx_point.int())
-        else:
-            device = neighbor_map.device
-            map_converted = torchsparse_backend.cpu_convert_map_forward(
-                neighbor_map.int().cpu(),
-                idx_batch.int().cpu(),
-                idx_point.int().cpu())
-            map_converted = map_converted.to(device)
-        nmap_offset = torch.sum(neighbor_map != -1, 1)
-        return map_converted.int().contiguous(), nmap_offset.int().contiguous()
 
-
-convert_neighbor_map_gpu = ConvertNeighborMap.apply
+def convert_neighbor_map_gpu(neighbor_map):
```

Comment:
We should probably rename these functions at some point, since both GPU and CPU are supported

Reply:
Thanks for the comment! I think the CPP functions for `convert_map_forward` will not be used elsewhere since it is tailored for `convert_neighbor_map_gpu`. However, the `insertion_forward` function might be useful in other applications (for example, SPVCNN and SPVNAS).

Reply:
Agree with @CCInc's comment. Maybe we can name it `squeeze_nmap`? Happy to discuss the naming.

Reply:
Good idea. I will go with `squeeze_nmap`.